Automattic/wp-calypso
# Components: modernize empty-content component and add to devdocs

This PR converts the `EmptyContent` component to ES6, moves the main file to index.jsx and does away with the package.json. I've also added it to Devdocs.

<img width="662" alt="screen shot 2017-05-26 at 17 56 24" src="https://cloud.githubusercontent.com/assets/17325/26502505/4c8b3af8-423d-11e7-9de5-a00c8ac2ce76.png">

### To test

Visit the new Devdocs page at:

http://calypso.localhost:3000/devdocs/design/empty-content

The sub-component, `no-sites-message`, is also affected. Visit a user account with no sites, and verify that you are shown the this message in My Sites:

<img width="725" alt="screen shot 2017-05-26 at 17 59 27" src="https://cloud.githubusercontent.com/assets/17325/26502495/483ca946-423d-11e7-9a61-c96201265f8c.png">
